tests/mapper/test_node_mappers.py

```diff
@@ -529,6 +545,29 @@ def test_hinnodemapper_manual_schema():
     )
 
 
+def test_hinnodemapper_zero_samples():
+    batch_size = 3
+    feature_sizes = {"t1": 1, "t2": 1}
+    G, nodes_type_1, nodes_type_2 = example_hin_3(feature_sizes)
+
+    mapper = HinSAGENodeGenerator(G, batch_size=batch_size, num_samples=[0, 0]).flow(
+        nodes_type_2
+    )
+
+    schema = G.create_graph_schema()
+    sampling_adj = schema.type_adjacency_list(["t2"], 2)
+
+    assert len(mapper) == 1
+
+    # Get a batch!
+    batch_feats, batch_targets = mapper[0]
+    assert len(batch_feats) == len(sampling_adj)
+
+    print(sampling_adj)
```

Comment:
Do we need this print statement here?

Reply:
Not really, although there are quite a few print statements we have left in the tests. Pytest suppresses these by default.